src/services/peopleSearch.ts

```diff
@@ -74,6 +74,29 @@ export type PeopleSearchQuery = {
   relationship_status?: string;
   involvement: string;
 };
+export interface SelectOption {
+  label: string;
+  value: string;
+}
+
+const getRenamedDepartments = async () => {
+  const dep = await getDepartments();
+  const mappedResults = <SelectOption[]>[];
+  dep.forEach((d) => {
+    let departmentName = { value: d, label: d };
+    if (/^Office of the /.test(d)) {
+      departmentName.label = d.replace(/^Office of the /, '') + ' (Office of the)';
+    } else if (/^Office of /.test(d)) {
+      departmentName.label = d.replace(/^Office of /, '') + ' (Office of)';
+    } else if (/^Center for /.test(d)) {
+      departmentName.label = d.replace(/^Center for /, '') + ' (Center for)';
+    } else if (/^Department of /.test(d)) {
+      departmentName.label = d.replace(/^Department of /, '') + ' (Department of)';
+    }
+    mappedResults.push(departmentName);
+  });
+  return mappedResults;
+};
```

Comment:
Circling back to the regular expression idea, I have tested this code in your branch and I know for a fact that it works:
```suggestion
/**
 * Match any department whose name starts with one of these prefixes:
 *  - "Office of (the) "
 *  - "Department of "
 *  - "Center for "
 */
const DepartmentPrefixRegex = /^(Office of(?: the)?|Department of|Center for) (.*)$/;

const getRenamedDepartments = async (): Promise<SelectOption[]> => {
  const departmentNames = await getDepartments();
  return departmentNames.map<SelectOption>((departmentName) => ({
    value: departmentName,
    label: departmentName.replace(DepartmentPrefixRegex, '$2 ($1)'),
  }));
};
```

I also think that, even despite the regular expression, it's clearer in intent and easier to read and maintain in the long run. That's just my opinion though.